armory/docker/images.py

```diff
@@ -19,23 +20,23 @@
 REPOSITORIES = tuple(x.split(":")[0] for x in ALL)
 
 
-def dev_version(tag):
+def parse_version(tag):
     """
     Return version.LooseVersion class for given version tag
```

Comment:
pkg_resources.version.Version

Reply:
For the base images - yes, I'll need to add those. I was individually targeting them - so I'll need to add those. Alternatively, I could just look over the entire `twosixarmory/` namespace.

Reply:
`pkg_resources.version.Version` does not exist (anymore?)

Reply:
found in extern